app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/OwnMoodsFragment.java

```diff
@@ -182,4 +184,12 @@ private void setUpFilterSpinner() {
         filterSpinner.setOnItemSelectedListener(new FilterEventHandlers());
     }
 
+    /*
+     * This function is used in tests to add a mood event into the data list
```

Comment:
I don't really like the existence of a function just for testing but it I also understand how useful it is here so we can leave it in.